docs/en/sql-reference/functions/array-functions.md

```diff
@@ -104,17 +104,560 @@ Can be optimized by enabling the [optimize_functions_to_subcolumns](../../operat
 
 Alias: `OCTET_LENGTH`
 
-## emptyArrayUInt8, emptyArrayUInt16, emptyArrayUInt32, emptyArrayUInt64
+## emptyArrayUInt8
 
-## emptyArrayInt8, emptyArrayInt16, emptyArrayInt32, emptyArrayInt64
+Accepts zero arguments and returns an empty UInt8 array.
 
-## emptyArrayFloat32, emptyArrayFloat64
+**Syntax**
+
+```sql
+emptyArrayUInt8()
+```
+
+**Arguments**
+
+None.
+
+**Returned value**
+
+An empty array.
+
+**Implementation details**
+
+None.
+
+**Examples**
+
+Query:
+
+```sql
+CREATE TABLE users (uid Int16, name String, age Int16, contacts Array(UInt8)) ENGINE=Memory;
+
+INSERT INTO users VALUES (1231, 'Alex', 33, emptyArrayUInt8());
+SELECT * FROM users;
+
+ALTER TABLE users UPDATE contacts = arrayPushBack(contacts, 255) WHERE uid = 1231;
```

Comment:
L. 139 - 140 and l. 147 are instructive but not really necessary to demonstrate functions `emptyArrayX`. For me as a casual reader, it would actually add mental overload. Suggest to remove that.

(same in the other documented functions)